src/core/Slider/Slider.stories.tsx

```diff
@@ -1,45 +1,65 @@
-import React, { ReactNode } from "react";
+import React from "react";
 import Slider from "./component.tsx";
+import Icon from "../Icon/component.tsx";
```

Comment:
These will cause merge conflicts now after the `set-up-14` merge, if you like I can resolve these

Reply:
Resolved them :grin: